platform/admin/ui/src/main/webapp/js/views/configuration/ConfigurationEdit.view.js

```diff
@@ -274,14 +296,23 @@ define([
                 });
             }
 
-            this.model.save();
-            wreqr.vent.trigger('sync');
-            wreqr.vent.trigger('poller:start');
-
-            _.defer(function() {
+            this.model.save().done(function () {
+                spinner.stop();
+                $('#config-modal').modal('hide');
```

Comment:
You should be able to use the jquery instance that is attached to the view instead of pulling in jquery yourself. Then it is automatically scoped the view instead of the entire DOM. Then you don't need the new "ID" attributes attached to the modals. You can simply do this.$('.modal-content') and it will retrieve your current modal. This will be true for all of these IDs that you added.